Sources/Moya/TargetType.swift

```diff
@@ -13,11 +13,8 @@ public protocol TargetType {
     /// The HTTP method used in the request.
     var method: Moya.Method { get }
 
-    /// The parameters to be encoded in the request.
-    var parameters: [String: Any]? { get }
-
-    /// The method used for parameter encoding.
-    var parameterEncoding: ParameterEncoding { get }
+    /// The default parameterEncoding for the `.encoded` RequestDataType case.
```

Comment:
Actually the extension didn't work correctly. I received the following error in Xcode 8.3:
`Static member 'default' cannot be used on protocol metatype 'ParameterEncoding.Protocol'`

I'm removing it now ... I think `JSONEncoding.default` and `URLEncoding.default` should be fine enough for now.